src/barplot.ts

```diff
@@ -325,7 +325,7 @@ export class Barplot {
     this.tooltip.text = initialText;
     this.tooltip.fadeIn();
 
-    const newText = `${initialText} ${data.value}`;
+    const newText = `${initialText} ${parseFloat(data.value).toFixed(4)}`;
```

Comment:
Great idea to reduce that long string. The right graph will later take input as strings, such as a ratio, represented as "1:1"  or percent's, with the symbol, e.g. "75%". With that in mind the parseFloat would remove those important symbols. The dev branch has a new property for `DataPoint`, called `nonNormalizedValue` which is used for the tooltip. Let's hold off on including this parse from this branch, and instead have the averaging methods also add that rounded value to the DataPoint as a `nonNormalizedValue`

Reply:
Good suggestion, going to reverse the parseFloat() line.